server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const { json } = require('body-parser')
+const { load, save } = require('./state')
+
+const app = express()
+
+// authorization
+app.use((req, res, next) => {
+  if (req.query.token !== process.env.TOKEN) {
+    res.status(401).send('Token invalid. Use a valid `token` in query parameter.')
+  } else {
+    next()
+  }
+})
+
+app.get('/:namespace/load', (req, res) => {
+  res.json(load(req.params.namespace))
+})
+
+app.put('/:namespace/save', json(), (req, res) => {
+  save(req.params.namespace, req.body)
+  res.json(load(req.params.namespace))
+})
+
+app.post('/:namespace/reset', (req, res) => {
+  save(req.params.namespace, {
+    entries: []
```

Comment:
What's the goal of the `entries` array?

Reply:
@sgimeno Just to make a point that the `reset` functionality resets the data to a predefined shape. There is no usage for `entries` at the moment.